cbdr/cortex
# COR-792: Pass 'Item Description' in Blog RSS Feed

## Purpose:
This PR adds the Description field to each Item in the Employer Blog RSS feed, so that Uberflip can pick up a description rather than trying to infer it from the posts' body copy.

## JIRA:
https://cb-content-enablement.atlassian.net/browse/COR-792

## Steps to Take On Prod
* You will have to manually migrate the RSS Decorator on Prod (you can just replace the db record with one from dev)

## Changes:
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * You will have to manually migrate the RSS Decorator on Prod (you can just replace the db record with one from dev)

* Library changes
  * N/A

* Side effects
  * N/A

## Screenshots
* Before
N/A

* After
N/A

## QA Links:
http://web.cortex-2.development.c66.me/

## How to Verify These Changes
* Specific pages to visit
  * N/A

* Steps to take
  * Create and *publish* a couple Employer Blogs
  * Visit http://web.cortex-2.development.c66.me/rss/v2/employer_blog
  * Ensure the published blogs appear with the `item/description` intact (*note* - not the `channel/description`). If they don't appear, try clearing the cache on the server.

* Responsive considerations
  * N/A

## Relevant PRs/Dependencies:
N/A

## Additional Information
N/A
